ddtrace/debugging/_probe/registry.py

```diff
@@ -50,9 +50,9 @@ def set_message(self, message):
 
 def _get_probe_location(probe):
     # type: (Probe) -> Optional[str]
-    if isinstance(probe, LineLocationDetails):
+    if isinstance(probe, LineLocationMixin):
         return probe.source_file
-    elif isinstance(probe, FunctionLocationDetails):
+    elif isinstance(probe, FunctionLocationMixin):
         return probe.module
     else:
         raise ValueError("Unsupported probe type: {}".format(type(probe)))
```

Comment:
Non-blocking for this PR: I wonder if we could define a `location` property on the location mixins that would return a tuple with the location information. Then we could drop this helper and simply do `probe.location[0]` to get either the source file or the module. `probe.location[1]` would then be either the line number or the function name.

Reply:
refactored - not sure if that what you wanted to see